models/DataObject/Data/QuantityValueRange.php

```diff
@@ -64,8 +64,21 @@ public function setMaximum(int|float|null $maximum): void
         $this->markMeDirty();
     }
 
-    public function getRange(int $step = 1): array
+    public function getRange(int|float $step = 1): array
```

Comment:
https://github.com/pimcore/pimcore/blob/823890c6f4e26e3e865716c2d1193f19c8c26762/models/DataObject/Data/QuantityValueRange.php#L72
`getValue()` should be also changed to accept float